pytests/func/test_flask.py

```diff
@@ -0,0 +1,321 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root
+# for license information.
+
+from __future__ import print_function, with_statement, absolute_import
+
+import os.path
+import pytest
+import platform
+
+from ..helpers.pattern import ANY
+from ..helpers.session import DebugSession
+from ..helpers.timeline import Event
+from ..helpers.webhelper import get_url_from_str, get_web_content
+from .testfiles.testroots import get_test_root
+
+
+FLASK1_ROOT = get_test_root('flask1')
+FLASK1_APP = os.path.join(FLASK1_ROOT, 'app.py')
+FLASK1_TEMPLATE = os.path.join(FLASK1_ROOT, 'templates', 'hello.html')
+
+
+def _wait_for_flask_link(debug_session):
```

Comment:
This looks identical to the Django version, so perhaps it should be in helpers as well?

(and then the same points apply here - can be done with a single `wait_for_next`)

Reply:
will switch all of them to connection wait